### PR TITLE
11.0 c1d920cbebf0de45e14b024e21aa44fc125256c0

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_disable_payment_restaurant/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                            font-weight: bold;
+                            color: #75526b;
+                            border: 3px dashed #75526b;
+                            float: left;
+                            padding: 4px 12px;
+                            -webkit-transform: rotate(-6deg);
+                            -o-transform: rotate(-6deg);
+                            -moz-transform: rotate(-6deg);
+                            -ms-transform: rotate(-6deg);">
+                    Tested on Odoo<br/>10.0 community
```
Comment (Ramil-Mukhametzyanov) [inline pos_disable_payment_restaurant/static/description/index.html:102]: 11.0 community

Comment (Ramil-Mukhametzyanov) [reply pos_disable_payment_restaurant/static/description/index.html:102]: It's OK, pos_disable_payment_restaurant is not ported yet.

File: pos_debt_notebook/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,7 @@
+`5.3.0`
+-------
+- **Improvement:** New Balance is amount for selected journal,  not for total Credits.
```
Comment (Ramil-Mukhametzyanov) [inline pos_debt_notebook/doc/changelog.rst:3]: checked :+1:

File: pos_disable_payment/doc/changelog.rst [hunk 3]
```diff
@@ -3,6 +3,12 @@
 Updates
 =======
 
+`3.7.0`
+-------
+
+**Improvement:** New option "Disable customer selection"
```
Comment (Ramil-Mukhametzyanov) [inline pos_disable_payment/doc/changelog.rst:9]: checked :+1:

File: pos_disable_payment/doc/changelog.rst [hunk 4]
```diff
@@ -3,6 +3,12 @@
...
+**Improvement:** Move the "Allow change Qty for kitchen orders" option to pos_disable_payment_restaurant module
```
Comment (Ramil-Mukhametzyanov) [inline pos_disable_payment/doc/changelog.rst:10]: checked :+1:

File: pos_keyboard/doc/changelog.rst [hunk 5]
```diff
@@ -1,7 +1,13 @@
...
+
+**Fix:** Error on clicking numbers and enter in selection popups
+
+
+`1.1.1`
+-------
 
-Updates
-=======
+**Fix:** Incorrect input data on password pop-up if the first clicked button was `backspace`
```
Comment (Ramil-Mukhametzyanov) [inline pos_keyboard/doc/changelog.rst:15]: checked :+1:

File: pos_partner_sync/doc/changelog.rst [hunk 6]
```diff
@@ -1,3 +1,9 @@
+`2.0.1`
+-------
+
+**Fix:** Client line data updating
```
Comment (Ramil-Mukhametzyanov) [inline pos_partner_sync/doc/changelog.rst:4]: checked :+1:

File: pos_partner_sync/doc/changelog.rst [hunk 7]
```diff
@@ -1,3 +1,9 @@
...
+**Fix:** Several updates within the one second were denied by built-in behavior
```
Comment (Ramil-Mukhametzyanov) [inline pos_partner_sync/doc/changelog.rst:5]: checked :+1:

File: pos_keyboard/doc/changelog.rst [hunk 8]
```diff
@@ -1,7 +1,13 @@
-.. _changelog:
+`1.1.2`
+-------
+
+**Fix:** Error on clicking numbers and enter in selection popups
```
Comment (Ramil-Mukhametzyanov) [inline pos_keyboard/doc/changelog.rst:9]: checked :-1:
Enter doesn't work. #985 
@KolushovAlexandr Remove the pos_keyboard change from the PR.

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]

File: pos_keyboard/doc/changelog.rst (hunk 8 above)
Comment (KolushovAlexandr) [reply pos_keyboard/doc/changelog.rst:9]: We realized that it is another error needed to be fixed in 10.0 at first

Comment (KolushovAlexandr) [reply pos_keyboard/doc/changelog.rst]: I'll remove this commit to update it and later merge in 11 again

File: pos_pin/doc/changelog.rst [hunk 9]
```diff
@@ -3,6 +3,11 @@
 Updates
 =======
 
+`1.2.1`
+-------
+
+- **Fix:** Cannot set property of undefined on some actions with password popup
```
Comment (Ramil-Mukhametzyanov) [inline pos_pin/doc/changelog.rst:9]: checked :-1:
The js files or py files associated with the pos_pin module in PR has not changed.

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]

File: pos_pin/doc/changelog.rst (hunk 9 above)
Comment (KolushovAlexandr) [reply pos_pin/doc/changelog.rst:9]: @Ramil-Mukhametzyanov it was fixed earlier as a part of the merge PR.

Comment (Ramil-Mukhametzyanov) [reply pos_pin/doc/changelog.rst:9]: OK

File: pos_orders_history/doc/changelog.rst [hunk 10]
```diff
@@ -6,6 +6,7 @@ Updates
 `1.2.1`
 -------
 
+**Fix:** Error related to load specific order after barcode scanning
 **Improvement:** Remove the "Update order list" button because the orders are updated instantly now
```
Comment (Ramil-Mukhametzyanov) [inline pos_orders_history/doc/changelog.rst:10]: checked :+1:

File: pos_orders_history_return/doc/changelog.rst [hunk 11]
```diff
@@ -3,6 +3,12 @@
 Updates
 =======
 
+`1.0.5`
+-------
+
+**Fix:** Error related to return a specific order after barcode scanning
```
Comment (Ramil-Mukhametzyanov) [inline pos_orders_history_return/doc/changelog.rst:9]: how to check this? #931 
@ilmir-k  @GabbasovDinar

File: pos_orders_history_return/doc/changelog.rst [hunk 12]
```diff
@@ -3,6 +3,12 @@
...
+**Fix:** Order duplication in case of poor connection in POS
```
Comment (Ramil-Mukhametzyanov) [inline pos_orders_history_return/doc/changelog.rst:10]: how to check this? #931 
@ilmir-k  @GabbasovDinar

File: pos_orders_history/doc/changelog.rst [hunk 13]
```diff
@@ -6,6 +6,7 @@ Updates
 `1.2.1`
 -------
 
+**Fix:** Error related to load specific order after barcode scanning
```
Comment (Ramil-Mukhametzyanov) [inline pos_orders_history/doc/changelog.rst:9]: how to check this? #931 
@ilmir-k  @GabbasovDinar

File: pos_orders_history_return/doc/changelog.rst (hunk 11 above)
Comment (ilmir-k) [reply pos_orders_history_return/doc/changelog.rst:9]: @Ramil-Mukhametzyanov 
load specific order by its barcode scanning and try to return it then

Comment (ilmir-k) [reply pos_orders_history_return/doc/changelog.rst:9]: @Ramil-Mukhametzyanov 
also try to make partial returns, i.e. 
* load specific order
* make a return of some products
* load the same order again
* return the remaining products

File: pos_orders_history_return/doc/changelog.rst (hunk 12 above)
Comment (ilmir-k) [reply pos_orders_history_return/doc/changelog.rst:10]: @Ramil-Mukhametzyanov 
emulate a poor connection and see if there are any duplicated orders after return or not

File: pos_orders_history/doc/changelog.rst (hunk 13 above)
Comment (ilmir-k) [reply pos_orders_history/doc/changelog.rst:9]: @Ramil-Mukhametzyanov 
Just load any order by barcode scanning and see results. It must be loaded without errors

Comment (Ramil-Mukhametzyanov) [reply pos_orders_history/doc/changelog.rst:9]: checked :+1:

File: pos_keyboard/doc/changelog.rst (hunk 8 above)
Comment (Ramil-Mukhametzyanov) [reply pos_keyboard/doc/changelog.rst:9]: **fixed here** #999

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
pos_order_history_return issues:
1. After entering the POS, if there is no paid order, the history screen cannot be opened.
2. There is no return button for orders in the history.

File: pos_keyboard/doc/changelog.rst [hunk 14]
```diff
@@ -1,7 +1,18 @@
-.. _changelog:
+`1.1.3`
+-------
+
+**Fix:** Enter did not work for some pop-ups
```
Comment (Ramil-Mukhametzyanov) [inline pos_keyboard/doc/changelog.rst:4]: checked :+1:

File: pos_orders_history_return/doc/changelog.rst (hunk 11 above)
Comment (Ramil-Mukhametzyanov) [reply pos_orders_history_return/doc/changelog.rst:9]: > @Ramil-Mukhametzyanov
> load specific order by its barcode scanning and try to return it then

checked :+1:

Comment (Ramil-Mukhametzyanov) [reply pos_orders_history_return/doc/changelog.rst:9]: > @Ramil-Mukhametzyanov
> also try to make partial returns, i.e.
> 
>     * load specific order
> 
>     * make a return of some products
> 
>     * load the same order again
> 
>     * return the remaining products

checked :+1:

File: pos_orders_history_return/doc/changelog.rst (hunk 12 above)
Comment (Ramil-Mukhametzyanov) [reply pos_orders_history_return/doc/changelog.rst:10]: checked :+1:

File: pos_multi_session/doc/changelog.rst [hunk 15]
```diff
@@ -1,3 +1,8 @@
+`4.2.8`
+-------
+
+- **Fix:** Paid order could appear again in POS in case of problems with connection within payment process
```
Comment (Ramil-Mukhametzyanov) [inline pos_multi_session/doc/changelog.rst:4]: checked :+1:

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Comment by yelizariev** [comment]
Травис штормит. Вернемся к ПР, когда травис наладится

**Comment by yelizariev** [comment]
rebase with upstream to fix travis

**Comment by yelizariev** [comment]
ok, seems that you just did something similar

File: pos_keyboard/README.rst [hunk 16]
```diff
@@ -1,25 +1,44 @@
...
+
+Sponsors
+--------
+* `IT-Projects LLC <https://it-projects.info>`__
+
+Maintainers
+-----------
+* `IT-Projects LLC <https://it-projects.info>`__
+
+      To get a guaranteed support you are kindly requested to purchase the module at `odoo apps store <https://apps.odoo.com/apps/modules/10.0/pos_keyboard/>`__.
```
Comment (yelizariev) [inline pos_keyboard/README.rst]: 11.0

**Review by yelizariev — APPROVED** [review]
Ok, but fix versions please

File: pos_pin/doc/changelog.rst [hunk 17]
```diff
@@ -1,7 +1,12 @@
-.. _changelog:
+`1.2.2`
+-------
+
+- **Fix:** Id of undefined error appeared when a pin was entered correctly
```
Comment (Ramil-Mukhametzyanov) [inline pos_pin/doc/changelog.rst:4]: checked :+1:

File: pos_multi_session_restaurant/doc/changelog.rst [hunk 18]
```diff
@@ -1,3 +1,7 @@
+`3.3.1`
+-------
+**Fix:** Property of undefined error might happen when restaurant multisession loses floors with opened orders in it
```
Comment (Ramil-Mukhametzyanov) [inline pos_multi_session_restaurant/doc/changelog.rst:3]: checked :+1:

File: pos_multi_session/doc/changelog.rst [hunk 19]
```diff
@@ -2,6 +2,7 @@
 -------
 
 - **Fix:** Paid order could appear again in POS in case of problems with connection within payment process
+- **Fix:** After restoring the connection, numbers of created and paid offline POS orders were not counted, that could lead to order loss in some cases
```
Comment (Ramil-Mukhametzyanov) [inline pos_multi_session/doc/changelog.rst:5]: checked :+1:

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by GabbasovDinar — APPROVED** [review]

**Pull request merged**